assets/javascripts/discourse/initializers/discourse-reactions.gjs

```diff
@@ -128,21 +128,25 @@ function initializeDiscourseReactions(api) {
 
         get label() {
           const count = this.notification.data.count;
-          const username = this.username;
+          const nameOrUsername = this.siteSettings.prioritize_full_name_in_ux
+            ? this.notification.data.display_name
+            : this.notification.data.display_username;
```

Comment:
```suggestion
            : this.username;
```

It currently uses `this.username` so we should keep it. They're practically the same, the only difference is that `this.username` has the `formatUsername` function applied to it whereas `this.notification.data.display_username` is the raw username.